src/test/java/github/weichware10/util/db/ConfigurationsTest.java

```diff
@@ -31,52 +31,50 @@ public ConfigurationsTest() {
                 dotenv.get("DB_SCHEMA"));
 
         CodeChartsConfiguration codeChartsConfiguration = new CodeChartsConfiguration(
-                "OBST",
-                new int[] { 20, 40 },
-                new long[] { 200, 400 },
-                true);
+                "OBST", new int[]{ 3, 5 }, new long[]{ 300, 500 },
+                false, true, true, 5,
+                15, -1, -1);
 
-        ccConfig = new Configuration("dunno yet", "Question?", "url", "intro", "outro",
+        ccConfig = new Configuration("dunno yet", "Question?", "url", "intro", "outro", true,
                 codeChartsConfiguration);
         String ccConfigId = dbClient.configurations.set(ccConfig);
-        ccConfig = new Configuration(ccConfigId, "Question?", "url", "intro", "outro",
+        ccConfig = new Configuration(ccConfigId, "Question?", "url", "intro", "outro", true,
                 codeChartsConfiguration);
 
-        ZoomMapsConfiguration zoomMapsConfiguration1 = new ZoomMapsConfiguration(4, 30, 30, true);
+        ZoomMapsConfiguration zoomMapsConfiguration1 = new ZoomMapsConfiguration(4, 30, 30);
 
-        zmConfig = new Configuration("dunno yet", "Question?", "url", "intro", "outro",
+        zmConfig = new Configuration("dunno yet", "Question?", "url", "intro", "outro", true,
                 zoomMapsConfiguration1);
         String zmConfigId = dbClient.configurations.set(zmConfig);
-        zmConfig = new Configuration(zmConfigId, "Question?", "url", "intro", "outro",
+        zmConfig = new Configuration(zmConfigId, "Question?", "url", "intro", "outro", true,
                 zoomMapsConfiguration1);
 
-        ZoomMapsConfiguration zoomMapsConfiguration2 = new ZoomMapsConfiguration(4, 40, 40, false);
+        ZoomMapsConfiguration zoomMapsConfiguration2 = new ZoomMapsConfiguration(4, 40, 40);
 
-        zmConfigWoTu = new Configuration("dunno yet", "Question?", "url", "intro", "outro",
+        zmConfigWoTu = new Configuration("dunno yet", "Question?", "url", "intro", "outro", true,
```

Comment:
Und bei CodeCharts wurde z.B. ein Tutorial boolean hinzugefügt

Reply:
das ist die allgemeine configuration, keine codechartsconfiguration

Reply:
es war vorher inkonsistent, wo die configuration hinzugefügt wurde und wo nicht, jetzt geschieht das immer in der allgemeinen configuration